docs/learn/features.md

```diff
@@ -0,0 +1,36 @@
+# Features
+Ceramic is a decentralized, open source platform for creating, hosting, and sharing streams of data. With Ceramic's permissionless data streaming network, you can store streams of information and ever-changing files directly on the decentralized web – and share updates with anyone in the world. This page describes the most important features of the protocol.
+
+### **Mutable streams**
+Store, edit, and update continuous [streams](./glossary.md#ipfs) of content or ever-changing files using stateful data structures on [IPFS](./glossary.md#ipfs) (specifically [IPLD](./glossary.md#ipld)). The data structure of your streams can be fully customized to your specific use cases and needs.
+
+### **Immutable naming**
+Reference streams by a persistent identifier, called a [StreamID](./glossary.md#streamid), instead of needing to keep track of IPFS [CIDs](./glossary.md#cid) that change every time your content does.
+
+### **Custom functions**
```

Comment:
I feel like I would re-order some of these.  "Custom functions" and "DID authentication" feel very low level and technical. Whereas "Sync & share" and "Limitless composability" feel more fundamental and generally useful/interesting, so I feel like those could come much earlier in the page

Reply:
fixed